server/services/prismic.js

```diff
@@ -1,8 +1,9 @@
 import Prismic from 'prismic.io';
 
-function removeParagraphs(string) {
-  return string.replace(/<\/?p>/g, '');
-}
+// TODO: DMC commented this because it wasn't being used
+// function removeParagraphs(string) {
+//   return string.replace(/<\/?p>/g, '');
+// }
```

Comment:
Maybe, aye. I've commented this out because I didn't write it (so didn't want to delete it in case it was an oversight to not use it), but the linter is complaining that it isn't used. And linting errors prevent non-forced pushes for this repo.